packages/forklift-console-plugin/tsconfig.json

```diff
@@ -1,22 +1,25 @@
 {
   "extends": "../../config/tsconfig.base.json",
 
-  "include": ["src/**/*", "*.ts"],
+  "include": ["./src/**/*", "./plugin-metadata.ts", "./plugin-extensions.ts"],
 
   "compilerOptions": {
-    "outDir": "dist",
-    "declarationDir": "dist/types",
+    "outDir": "./dist",
```

Comment:
The `declarationDir` was nice simply because it keeps the generated `*.d.ts(.map)?` files under `./dist/types/` .  Those emitted files aren't needed for the console-plugin AFAICT.  The normal source maps are already included in the bundled chunks.

Reply:
yes I removed it because the plugin 8s not a lib ... and it's packaged be webpack script that will ignore this files
I have no strong feelings about this line